dev/com.ibm.ws.channelfw/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -54,4 +54,10 @@ tcp.hostNameIncludeList=Host name include list
 tcp.hostNameIncludeList.desc=A comma-separated list of host names that are allowed to make inbound connections on this endpoint. Host names are not case-sensitive and can start with an asterisk, which is used as a wildcard character. However, asterisks cannot be elsewhere in the host name. For example, *.abc.com is valid, but *.abc.* is not valid. 
 
 tcp.hostNameExcludeList=Host name exclude list
-tcp.hostNameExcludeList.desc=A comma-separated list of host names that are not allowed to make inbound connections on this endpoint. Host names are not case-sensitive and can start with an asterisk, which is used as a wildcard character. However, asterisks cannot be elsewhere in the host name. For example, *.abc.com is valid, but *.abc.* is not valid.
\ No newline at end of file
+tcp.hostNameExcludeList.desc=A comma-separated list of host names that are not allowed to make inbound connections on this endpoint. Host names are not case-sensitive and can start with an asterisk, which is used as a wildcard character. However, asterisks cannot be elsewhere in the host name. For example, *.abc.com is valid, but *.abc.* is not valid.
+
+tcp.acceptThread=Accept thread
+tcp.acceptThread.desc=If true, then listening ports do not share the same thread for accepting connections. Otherwise, they share the same thread.
+
+tcp.waitToAccept=Wait to accept
+tcp.waitToAccept.desc=Closes connections until the server completely starts up. This is to be used with the acceptThread tcpOption set to true.
```

Comment:
Be specific. What closes the connection? If knowing what closes the connections isn't important, then consider writing:
Connections are closed until the server completely starts. Use this property with the acceptThread tcpOption set to true.